taskcluster/kinds/bicleaner/kind.yml

```diff
@@ -33,7 +33,7 @@ tasks:
                     - pipeline/bicleaner/requirements/bicleaner-ai.txt
                 from-parameters:
                     bicleaner_threshold:
-                        - training_config.experiment.bicleaner.dataset-thresholds.{dataset}
+                        - training_config.experiment.bicleaner.dataset-thresholds.{provider}_{dataset_sanitized}
```

Comment:
Were we ignoring bicleaner thresholds (other than the default) because of this previously? (Based on the commit that added this it looks like maybe not? I just wanted to double check...)

Reply:
Only for the datasets with "." in the names, but we didn't use the thresholds for such datasets, so no.